guides/v2.1/contributor-guide/templates/tutorial-template-last.md

```diff
@@ -1,20 +1,50 @@
 ---
 layout: tutorial
-group: contributor
-subgroup:
 title: Step X. Finish the tutorial
 subtitle: Sample template tutorial
-menu_title:
 menu_order: 2
-level3_menu_node:
 level3_subgroup: sample-tutorial
+return_to:
+   title: Code Contributions
+   url: contributor-guide/contributing.html
+
+
 ---
 
-The `title` and `subtitle` parameters provide the top-level headings on this page. Add the content as you see fit.
+The last page of tutorial should acknowledge the reader has completed the tutorial and provide links to related information. Otherwise, add the content as you see fit.
+
+## Metadata parameters
+
+The {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters for the last page of a tutorial are the same as a middle page.
+
+Parameter | Description
+--- | ---
+`layout:` | Must be `tutorial`.
+`title:` | The title of the topic. This value should be in the form _Step X. Perform this task_. (For example, `Step 1. Configure the Store`)
+`menu_title:`| Optional. Use this parameter only if you don't want the title in the left navigation to match the topic title.
+`subtitle` | The title of the tutorial. This value should match the `title` value in the index file.
+`level3_subgroup:` | A group name that binds the tutorial topics together. This value must be specified on each topic in the tutorial.
+`menu_order:` | The sequence number of the topic. For the index page in a multi-step tutorial, set this 0. Otherwise set it to the step number.
+`return_to:` | Defines the tutorial's parent page. The parent page title is displayed in the left navigation above the tuturial steps. Do not specify a value for the `return_to:` parameter. Instead, specify values for these second-level parameters:<br/><br/>`title:` The title of the parent topic<br/><br/>`url:` The URL of the parent topic
+`ee_only:` | Optional. If set to `true`, graphics/cues indicating that the article applies to EE are displayed on devdocs.
```

Comment:
Replace obsolete reference to "EE" with variable `{{site.data.var.ee}}`.

Also, apply style attribute to table: `{:style="table-layout:auto;"}`